app/models/ability.rb

```diff
@@ -6,8 +6,10 @@ def initialize(user) # rubocop:disable all
       case user.view_mode
       when 'superuser'
         can :manage, :all
+        can :view_detailed, EquipmentModel
```

Comment:
You don't need to specify this for superusers or admins since `can :manage, :all` gives them all abilities by default.